packages/compose/src/hooks/use-focus-on-mount/index.js

```diff
@@ -7,7 +7,7 @@ import { focus } from '@wordpress/dom';
 /**
  * Hook used to focus the first tabbable element on mount.
  *
- * @param {boolean | 'firstElement'} focusOnMount Focus on mount mode.
+ * @param {boolean | 'firstElement' | Function} focusOnMount Focus on mount mode.
```

Comment:
I wonder if we should also add a CHANGELOG entry to the compose package for the changes to the `useFocusOnMount` hook?

Reply:
✅ Good idea. Added.